.spelling

```diff
@@ -356,3 +357,31 @@ HelloWorldABCIAbstractRound
 PrintMessageBehaviour
  - docs/agent_services_context.md
 
+Fetch.ai
+Hosseini
+Bukva
+pbukva
+Jiri
+Vestfal
+MissingNO57
+ejfitzgerald
+Panasevych
+Panasevychol
+Lokman
+Rahmani
+lrahmani
+Turchenkov
+solarw
+eriknw
+Rocklin
+mrocklin
+Buitinck
+larsmans
+Thouis
+thouis
+scoder
+Phillip
+cpcloud
+Jevnik
+llllllllll
+3-Clause
```

Comment:
????

Reply:
Names from `AUTHORS.md`, I suggest we keep them temporary if this is an issue. We can deal with this later

Reply:
Ugh this is not the right authors...you got them from some other repo! https://github.com/valory-xyz/open-autonomy/blob/main/AUTHORS.md